src/swell/deployment/platforms/generic/suite_questions.yaml

```
@@ -2,7 +2,7 @@ experiment_id:
   default_value: swell-{{suite_to_run}}
 
 experiment_root:
-  default_value: /home/${USER}/SwellExperiments
+  default_value: $HOME/SwellExperiments
 
 r2d2_local_path:
   default_value: /home/${USER}/R2D2DataStore/Local
```

For consistency with `experiment_root: default_value` and a potentially changing /home filesystem: 
```suggestion
  default_value: $HOME/R2D2DataStore/Local
```

---

Thanks for the suggestion.